CHANGES.md

```diff
@@ -1,5 +1,22 @@
 # Changes
 
+## edge-21.3.4
+
+This edge release is a release candidate for stable 2.10.1 and fixes
+some issues around publishing of CLI binary for Apple Silicon
+M1 Chips. This release also includes some improvements to the
+destination, and the CLI.
+
+* Updated the IP Watcher in destination to ignore pods in "Terminating" state
+  (thanks @Wenliang-CHEN!)
+* Added `installNamespace` toggle in the jaeger extension's install.
+  (thanks @jijeesh!)
+* Updated `healthcheck` pkg to have `hintBaseURL` configurable, useful
+  for external extensions using that pkg.
+* Added multi-arch support for RabbitMQ integration tests (thanks @barkardk!)
+* Fixed an issue where MacOS binary was not being built.
+* Added darwin-arm64 CLI binary to list of uploads in `release.yml`
```

Comment:
None of this is really user-facing, right? We already have darwin-amd64 binaries on the last edge release.

```suggestion
```

Reply:
okay removed them.